### PR TITLE
Update regexp-tree to 0.0.83

### PR DESCRIPTION
**Pull request by jhnns**
Fixes an unexpected behavior where regexp-tree produced a longer regexp than the original one.

See https://github.com/DmitrySoshnikov/regexp-tree/pull/151

Since this is a potential breaking change, you should do a major version bump when publishing the module :)

### CODE REVIEW

File: tests/lib/rules/optimize-regex.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ ruleTester.run('optimize-regex', rule, {
     'var foo = /bar/mig',
     'var foo = /\\/\\./',
     'var foo = /[/\\\\]$/',
+    'var re = /foo/'
```
Comment (BrainMaestro) [inline tests/lib/rules/optimize-regex.js]: could you add the trailing comma

File: tests/lib/rules/optimize-regex.js [hunk 2]
```diff
@@ -32,14 +33,6 @@ ruleTester.run('optimize-regex', rule, {
...
-      code: 'var re = /foo/',
-      output: 'var re = /fo{2}/',
-      errors: [
-        {
-          message: '/foo/ can be optimized to /fo{2}/',
-          type: 'Literal',
-        },
-      ],
-    },
+    }
```
Comment (BrainMaestro) [inline tests/lib/rules/optimize-regex.js]: ditto

**Comment by jhnns** [comment]
I've added the trailing commas.

I think it is a breaking change, because code that used to be ok can be an error now. If someone uses the plugin and configured it to be an error, the linting test might suddenly break without installing a major version bump.

**Comment by BrainMaestro** [comment]
Hey @jhnns sorry about this, but this PR somehow slipped through the cracks and I completely forgot about it. I have set up a bot to automatically update the dependencies though. Thanks for the PR, I'll close this now.

**Comment by jhnns** [comment]
No worries 👍

**Pull request closed without merging**